opendevin/_logging.py

```diff
@@ -0,0 +1,121 @@
+import logging, os, sys
```

Comment:
Is the underscore prefix for the filename typical of python? I haven't seen that before

Reply:
LiteLLM is doing it, among our stuff, and I stole it. But it's very uncommon. PEP 8 doesn't mention it directly, and at most suggests that it's native code (c/c++). I'll change it.